### ISSUE TITLE
Dynamic accessible name on dropdown

### ISSUE THREAD

**Issue by nilamf** [WebAccessibilityTools/CCA#31]
The dropdown's accessible name updates dynamically based on the selected value or state. Since the dropdown lacks a visible label, this constant change can be disorienting for users relying on assistive technology.

I am not sure if this is intentional, but it deviates from standard accessibility patterns where the name (label) remains static while the value changes.

Version: v0.1.8 for MAC OS.
Test device: MacBook Pro macOS 15.7.5 (24G624)

<img width="113" height="34" alt="Image" src="https://github.com/user-attachments/assets/879ef612-252c-45e0-babd-d3a699b18903" />

**Comment by ferllings** [comment]
That's not intentional.
It was probably late when I did that :)

**Comment by ferllings** [comment]
@nilamf In fact the static label is properly announced for me: "Slider display mode  combo box  Dynamic  collapsed"
Does that not work for you?

**Comment by nilamf** [comment]
Sorry, I missed that! I think I got confused since I didn't see a visible label and I didn't expect the name 'Slider display mode'. So sorry that I wasted your time!

**Comment by ferllings** [comment]
No worries, thanks for the feedback
